src/main/resources/db/migrationv2/V2_13__update_juror_add_response_entered_column.sql

```diff
@@ -0,0 +1,8 @@
+ALTER TABLE juror_mod.juror
+  ADD COLUMN response_entered BOOLEAN DEFAULT FALSE;
+
+UPDATE juror_mod.juror j
```

Comment:
this is going to cause deployment issues, need to take this bit out, do the update as a separate release step

Reply:
taken out and will be run separately on release